code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -282,12 +282,22 @@
 		if(!LH.target)
 			var/datum/objective/A = new
 			A.owner = user.mind
+			var/list/datum/team/teams = list()
+			for(var/datum/antagonist/antag in user.mind.antag_datums)
+				var/datum/team/T = antag.get_team()
+				if(T)
+					teams |= T
 			var/list/targets = list()
 			for(var/i in 0 to 3)
 				var/datum/mind/targeted =  A.find_target()//easy way, i dont feel like copy pasting that entire block of code
+				var/is_teammate = FALSE
+				for(var/datum/team/T in teams)
+					if(targeted in T.members)
+						is_teammate = TRUE
+						break
 				if(!targeted)
 					break
-				targets["[targeted.current.real_name] the [targeted.assigned_role]"] = targeted.current
+				targets["[targeted.current.real_name] the [targeted.assigned_role][is_teammate ? " (ally)" : ""]"] = targeted.current
```

Comment:
Why don't we just change the target selection algorithm to prioritize people not on your team first, only giving you people on your team when all other options are exhausted?

Reply:
> Why don't we just change the target selection algorithm to prioritize people not on your team first, only giving you people on your team when all other options are exhausted?

if you do this, please keep the (ally) thing, for those rare scenarios where there aren't enough non-allied potential targets to fill out a whole list.